src/app/views/communities/view/overview/edit/games/games.ts

```diff
@@ -0,0 +1,139 @@
+import Component from 'vue-class-component';
+import { Emit } from 'vue-property-decorator';
+import { arrayRemove } from '../../../../../../../utils/array';
+import { Api } from '../../../../../../../_common/api/api.service';
+import AppCardListAdd from '../../../../../../../_common/card/list/add/add.vue';
+import AppCardListDraggable from '../../../../../../../_common/card/list/draggable/draggable.vue';
+import AppCardListItem from '../../../../../../../_common/card/list/item/item.vue';
+import AppCardList from '../../../../../../../_common/card/list/list.vue';
+import { Game } from '../../../../../../../_common/game/game.model';
+import AppGameThumbnailImg from '../../../../../../../_common/game/thumbnail-img/thumbnail-img.vue';
+import { Growls } from '../../../../../../../_common/growls/growls.service';
+import {
+	BaseRouteComponent,
+	RouteResolver,
+} from '../../../../../../../_common/route/route-component';
+import { AppTooltip } from '../../../../../../../_common/tooltip/tooltip';
+import { CommunityLinkGameModal } from '../../../../../../components/community/link-game-modal/link-game-modal.service';
+import { AppCommunityPerms } from '../../../../../../components/community/perms/perms';
+import { RouteStore, RouteStoreModule } from '../edit.store';
+
+@Component({
+	name: 'RouteCommunitiesViewEditGames',
+	components: {
+		AppCommunityPerms,
+		AppCardList,
+		AppCardListDraggable,
+		AppCardListItem,
+		AppCardListAdd,
+		AppGameThumbnailImg,
+	},
+	directives: {
+		AppTooltip,
+	},
+})
+@RouteResolver({
+	deps: { params: ['id'] },
+	resolver: ({ route }) => Api.sendRequest('/web/dash/communities/games/' + route.params.id),
```

Comment:
We can avoid this request entirely if we have all games gathered when a collaborator views the community.